src/EditorFeatures/Test2/GoToDefinition/CSharpGoToDefinitionTests.vb

```diff
@@ -1577,6 +1577,40 @@ class D
 
 #End Region
 
+        <WpfFact>
+        Public Async Function TestCallingConv() As Task
+            Dim workspace =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+unsafe
+{
+    delegate* unmanaged[$$Cdecl]&lt;void&gt; f1;
+}
+        </Document>
+    </Project>
+</Workspace>
+
+            Await TestAsync(workspace)
```

Comment:
this doesn't actually seem to demonstrate it working :)

Reply:
@CyrusNajmabadi It does. The test was actually failing before implementing my fix.

Basically, `TestAsync` has a bool optional parameter that specifies whether a successful navigation happened or not. The parameter is true by default.